tools/test/.pylintrc

```diff
@@ -3,14 +3,22 @@
 # A comma-separated list of package or module names from where C extensions may
 # be loaded. Extensions are loading into the active Python interpreter and may
 # run arbitrary code.
+extension-pkg-allow-list=
```

Comment:
These changes feel like more than "allow for
longer strings to allow for the additional artifacts in the storage
tests." And since they wont be needed if we move the test data to actual files maybe its not necessary?